src/test_workflow/bwc_test/bwc_test_suite_opensearch.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Nit: this class is a whole lot of stuff for effectively two lines of differentiated behavior.  I don't think there is an easier way but food for thought